src/pages/pages.css

```diff
@@ -0,0 +1,4 @@
+h1 {
+  padding: 20px;
+  text-align: center;
+}
```

Comment:
Is the idea with `pages.css` that this will contain CSS that’s used across the different views and that we might have additional files such as `Map.css` for styles specific to that view?